cancermuts/datasources.py

```diff
@@ -533,10 +533,8 @@ def __init__(self, database_files=None, database_encoding=None):
 
         dataframes = []
 
-        if database_files is None:
-            database_dir   = '/data/databases/cosmic'
-            databases      = [  'CosmicMutantExport' ]
-            self._database_files = [os.path.join(database_dir, i)+'.tsv' for i in databases]
+        if isinstance(database_files, str):
+            self._database_files = list(database_files)
         else:
             self._database_files = database_files
```

Comment:
also add check on the fact that it is an interable with e.g. `hasattr(database_files, '__iter__')` as we discussed yesterday and raise an exception if it's not (the argument should be either a string or an iterable)